scripts/evaluate_release_tag.py

```diff
@@ -63,9 +72,11 @@ def should_build_docs(tag: Version) -> bool:
     elif tag >= latest_version:
```

Comment:
Shouldn't this update to `tag >= latest_version_non_beta`?

Reply:
Yes, simplified the function now